src/macros.rst

```diff
@@ -412,10 +425,11 @@ the :t:`token` stream produced from the related :s:`EnumDeclaration`,
 A :t:`derive macro` adds all its declared :t:`[derive helper attribute]s` into
 the :t:`abstract data type scope` of the :t:`abstract data type` the
 :t:`attribute` is attached to.
+IS THIS RELEVANT NOW? WHICH SCOPE?
```

Comment:
Ah, right this is a thing. These are basically visible for attributes on the whole ADT.
```rs
#[derive(MyDerive)]
#[helper_visible_here]
enum Foo {
   #[helper_visible_here]
   Variant {
       #[helper_visible_here]
       field: ()
   }
}

```